scripts/release_scripts/run_macaron.sh

```diff
@@ -598,7 +599,21 @@ else
     fi
 fi
 
+# Handle tokens.
+set +u
+echo "" > ${TOKEN_FILE}
+{
+    echo "GITHUB_TOKEN=${GITHUB_TOKEN}" >> ${TOKEN_FILE}
+    echo "MCN_GITLAB_TOKEN=${MCN_GITLAB_TOKEN}" >> ${TOKEN_FILE}
+    echo "MCN_SELF_HOSTED_GITLAB_TOKEN=${MCN_SELF_HOSTED_GITLAB_TOKEN}"
+} >> ${TOKEN_FILE}
```

Comment:
Why are the first two echoes redirected directly into the file, and then the last one is echoed via the containing block redirect? I mean, it works, but the inconsistency just looks a bit odd.

Reply:
Interesting, I agree that we could echo the third one directly into the file :thinking: 
However, I merely move this block location within `run_macaron.sh` without changing its original content, therefore I am not sure of the reason behind it. 
This block was originally added in https://github.com/oracle/macaron/pull/605

Reply:
Addressed in https://github.com/oracle/macaron/pull/780/commits/a6a0d98f416f013000d8d14b210158d4b62a7f08